### PR TITLE
RWA-1520 remove unneeded jobs

### PR DESCRIPTION
**Pull request by vgulapally**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-1520

### Change description ###
Remove R1 elements and unneeded jobs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by dharmendrak — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SearchCasesConsumerTest.java [hunk 1]
```diff
@@ -1,63 +0,0 @@
...
-            .status(HttpStatus.SC_OK)
-            .body(buildSearchResultDsl(CASE_ID))
-            .toPact();
-    }
-
-    @Test
-    @PactTestFor(pactMethod = "searchCases")
-    public void verifySearchCases() {
-
-        ElasticSearchCaseList searchResult = ccdClient.searchCases(AUTH_TOKEN,
```
Comment (burale) [inline src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SearchCasesConsumerTest.java:58]: This is used currently in ElasticSearchCaseRetrieverService

        return ccdClient.searchCases(
            systemUserIdamToken.generate(),
            param.getServiceAuthentication(),
            "Asylum",
            ResourceUtility.getResource(param.getResourceEnum())
        );

This pact test needs to be reintroduced.

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/StartEventForCaseWorkerConsumerTest.java [hunk 2]
```diff
@@ -1,109 +0,0 @@
...
-            .status(200)
-            .body(buildStartEventForCaseWorkerPactDsl(START_APPEAL))
-            .toPact();
-    }
-
-    @Test
-    @PactTestFor(pactMethod = "startEventForCaseWorker")
-    public void verifyStartEventForCaseworker() throws JSONException {
-
-        final StartEventResponse startEventResponse = coreCaseDataApi.startEventForCaseWorker(
```
Comment (burale) [inline src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/StartEventForCaseWorkerConsumerTest.java:56]: This is used currently in CaseManagementDataService

        StartEventResponse startEventResponse = coreCaseDataApi.startEventForCaseWorker(
            parameter.getUserAuthorization(),
            parameter.getServiceAuthorization(),
            parameter.getUserId(),
            "IA",
            "Asylum",
            parameter.getCaseId(),
            EVENT_ID
        );

This pact test needs to be reintroduced.

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/StartForCaseWorkerConsumerTest.java [hunk 3]
```diff
@@ -1,82 +0,0 @@
...
-            .status(200)
-            .body(buildStartForCaseWorkerPactDsl(START_APPEAL))
-            .toPact();
-    }
-
-    @Test
-    @PactTestFor(pactMethod = "startForCaseWorker")
-    public void verifyStartEventForCaseworker() throws JSONException {
-
-        StartEventResponse startEventResponse = coreCaseDataApi.startForCaseworker(
```
Comment (burale) [inline src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/StartForCaseWorkerConsumerTest.java:56]: This is used currently in GivensBuilder -> CoreCaseDataApi
        StartEventResponse startCase = coreCaseDataApi.startForCaseworker(
            userToken,
            serviceToken,
            userInfo.getUid(),
            "IA",
            "Asylum",
            "startAppeal"
        );

This pact test needs to be reintroduced.

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SubmitEventForCaseWorkerConsumerTest.java [hunk 4]
```diff
@@ -1,113 +0,0 @@
...
-            .status(201)
-            .body(buildSubmitForCaseWorkedPactDsl(CASE_ID))
-            .toPact();
-    }
-
-    @Test
-    @PactTestFor(pactMethod = "submitEventForCaseWorker")
-    public void verifySubmitEventForCaseworker() throws Exception {
-
-        final CaseDetails caseDetails = coreCaseDataApi.submitEventForCaseWorker(
```
Comment (burale) [inline src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SubmitEventForCaseWorkerConsumerTest.java:60]: This is used currently in GivensBuilder

        coreCaseDataApi.submitEventForCaseWorker(
            userToken,
            serviceToken,
            userInfo.getUid(),
            "IA",
            "Asylum",
            caseDetails.getId().toString(),
            true,
            submitCaseDataContent
        );

This pact test needs to be reintroduced. This is used in functional test.

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/StartForCaseWorkerConsumerTest.java (hunk 3 above)
Comment (vgulapally) [reply src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/StartForCaseWorkerConsumerTest.java]: okay got it, i can retain them

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SubmitEventForCaseWorkerConsumerTest.java (hunk 4 above)
Comment (vgulapally) [reply src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SubmitEventForCaseWorkerConsumerTest.java]: okay got it, i can retain them

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SubmitForCaseWorkerConsumerTest.java [hunk 5]
```diff
@@ -1,117 +0,0 @@
...
-            .matchHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
-            .toPact();
-    }
-
-    @Test
-    @PactTestFor(pactMethod = "submitCaseWorkerDetails")
-    public void submitForCaseWorker() throws Exception {
-
-        caseDataContent = createCaseDataContent(SUBMIT_APPEAL, caseDetailsMap);
-        CaseDetails caseDetailsResponse = coreCaseDataApi.submitForCaseworker(
```
[the remark this answers is not in the document]
Comment (vgulapally) [reply src/contractTest/java/uk/gov/hmcts/reform/wataskmonitor/consumer/ccd/SubmitForCaseWorkerConsumerTest.java]: okay got it, i can retain them

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**